fdb-extensions/src/test/java/com/apple/test/BooleanArgumentsProvider.java

```diff
@@ -25,25 +25,29 @@
 import org.junit.jupiter.params.provider.ArgumentsProvider;
 import org.junit.jupiter.params.support.AnnotationConsumer;
 
-import javax.annotation.Nonnull;
 import java.util.Arrays;
-import java.util.List;
 import java.util.stream.Stream;
 
 /**
  * Argument provider for the {@link BooleanSource} annotation for providing booleans to parameterized
- * tests. Regardless of the source or context, this always returns {@code false} and {@code true} in that order.
+ * tests. Regardless of the source or context, this always returns {@code false} and {@code true} in that order for each
```

Comment:
@ohadzeliger I mentioned in the commit message, but the original javadoc stated the order these would come back. I don't see a reason to change that behavior here, so I changed `booleans` to return in the same order.